lib/Dialect/SV/SVOps.cpp

```diff
@@ -166,6 +166,33 @@ void IfDefOp::build(OpBuilder &odsBuilder, OperationState &result,
   }
 }
 
+static bool isEmptyBlockExceptForTerminator(Block *block) {
+  if (!block)
+    return true;
+  return llvm::all_of(block->without_terminator(),
+                      [](auto &op) { return false; });
```

Comment:
For this, you can do something like:
`return block->empty() || block->front().hasTrait<OpTrait::IsTerminator>();`